EmailService/EmailService.Data/SmtpSettingsRepository.cs

```diff
@@ -63,5 +63,10 @@ public async Task<DbModuleSetting> GetAsync()
     {
       return await _provider.ModuleSettings.FirstOrDefaultAsync();
     }
+
+    public async Task<bool> DoesExistAnyAsync()
+    {
+      return !(_provider.ModuleSettings.FirstAsync() is null);
```

Comment:
This can be easier "_provider.ModuleSettings.FirstAsync() is not null;"

Reply:
changed. thanks